app/src/main/java/com/benoitquenaudon/daggerbundleinjectiondemo/injection/module/OutputActivityModule.java

```diff
@@ -1,7 +1,16 @@
 package com.benoitquenaudon.daggerbundleinjectiondemo.injection.module;
 
+import com.benoitquenaudon.daggerbundleinjectiondemo.BundleService;
+import com.benoitquenaudon.daggerbundleinjectiondemo.injection.SomeInput;
+import com.benoitquenaudon.daggerbundleinjectiondemo.injection.scope.ScopeActivity;
 import com.benoitquenaudon.daggerbundleinjectiondemo.ui.output.OutputActivity;
 import dagger.Module;
+import dagger.Provides;
 
-@Module abstract class OutputActivityModule implements BaseAppCompatActivityModule<OutputActivity> {
+@Module public abstract class OutputActivityModule implements BaseAppCompatActivityModule<OutputActivity> {
+  public static final String EXTRA_SOME_INPUT = "com.benoitquenaudon.demo.SOME_INPUT";
```

Comment:
> I'm not sure what you mean by causing duplicates, would you mind explaining? When you have a chance.

What if an activity A can be accessed from both activity R and activity S. If both R and S use the same key for an extra that does not have the same meaning, that would make things complicated inside A and hard to debug I think.